app/models/user.rb

```diff
@@ -236,6 +236,16 @@ def send_devise_notification(notification, *args)
     devise_mailer.send(notification, self, *args).deliver_later
   end
 
+  # check if user is active or not
+  def active_for_authentication?
+    super && self.active?
+  end
+
+  # message to the user that is not allowed to login
+  def inactive_message
+    "You are not allowed to log in!"
```

Comment:
I think @muescha is saying to use an i18n string instead of hard coding the message